modules/ROOT/pages/design-publish.adoc

```diff
@@ -0,0 +1,47 @@
+= Publish an API Specification
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+When you want to share your API specification with other developers, you can publish it to Anypoint Exchange. Exchange creates a page that presents the project for anyone in your business group to view and download the project from.
+
+.The page that Exchange creates for a published project includes a left pane for viewing the specification's documentation, a middle pane for viewing a description of the specification, and a right pane for viewing metadata about the specification.
+image:apid-example-page-in-exchange.png[An example page for a specification published in Exchange]
+
+<1> The left pane, which you can navigate to view the documentation. The navigation is generated from the documentation in the specification.
+<2> The middle pane, which describes the specification. You write the description in Exchange after publishing the specification.
+<3> The right pane, which displays metadata about the specification.
+
+== Procedure
```

Comment:
We should explain that one of the steps is to `set the root-file`.
For projects that were created before this release, it is possible that the root-file is not set. In these cases when the Project is opened the following message is displayed:
![2020-01-31_11-16-05](https://user-images.githubusercontent.com/12468128/73546151-779ff980-441b-11ea-9c89-eefbc4797bd5.png)

If the root-file is not set, it cannot be published to Exchange.

Reply:
@iamerico , I've added this note to the doc about root files:

> The first time that you open in the text editor a project that was created before the February 8, 2020 release of API Designer, you are prompted to set a root file.